### DISCUSSION TITLE
Arduino Nano RP2040 Connect does not know the onboard LED (LED_BUILTIN)

### DISCUSSION THREAD

**Discussion by hybotix** [micropython/micropython#13320] [RP2040 / Pico]
>>> 
MPY: soft reboot
MicroPython v1.21.0 on 2023-10-06; Arduino Nano RP2040 Connect with RP2040
Type "help()" for more information.
>>> from machine import Pin
>>> led = Pin("LED_BUILTIN", Pin.OUT)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: unknown named pin "LED_BUILTIN"

**Comment by robert-hh** [comment]
The on-board LEDs at the Arduino Nano RP2040 Connect are called LED, LED_RED, LED_GREEN and LED_BLUE. LED or PIN names are not the same across boards. The Pin names can be seen with:

```
import machine
print(dir(machine.Pin.board))
```

**Comment by sebromero** [comment]
@robert-hh I'm working on a PR to add the missing pin names. Will be done shortly.

**Comment by partha235** [comment]
try with 
led=Pin("LED",Pin.OUT)

**Comment by partha235** [comment]
what was current stage
